extensions/keep-frontmost/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "keep-frontmost",
+  "title": "Keep Frontmost",
```

Comment:
Can we think of a better name, it's kinda not saying what it does

Reply:
What do you think about SingleFocus ?

Reply:
That could work as title for sure